docs/adguard-for-safari/installation.md

```diff
@@ -31,25 +35,18 @@ AdGuard for Safari is a free app presented on the App Store. To install it on yo
 
     ![Extensions checked](https://cdn.adtidy.org/content/Kb/ad_blocker/safari/adguard-for-safari-extensions-checked.png)
 
-1. To make AdGuard take effects on websites, the *Content Blockers* setting must be enabled. You can either enable for all websites or select the ones you want.
+1. In order for AdGuard to have an effect on websites, the *Extensions* setting in Safari must be enabled. Open Safari → *Settings* → *Websites* → *Extensions*. If you want to enable AdGuard on every website, set *For other websites* to *Allow*.
```

Comment:
For AdGuard to work on websites, the *Extensions* setting in Safari must be enabled.